extensions/vscode/webviews/homeView/src/components/views/projectFiles/tooltips.ts

```diff
@@ -1,14 +1,18 @@
 import { ContentRecordFile, FileMatchSource } from "../../../../../../src/api";
 
-export function includedFileTooltip(file: ContentRecordFile) {
+export function includedFileTooltip(
+  file: Pick<ContentRecordFile, "rel" | "reason">,
```

Comment:
These changes allowed for easy usage for `ContentRecordFile` and the `FlatFile` type.